lib/improver/cli/orographic_enhancement.py

```diff
@@ -71,77 +64,20 @@ def load_and_extract(cube_filepath, height_value, units):
-                        help='Full path to input NetCDF file of wind speed on '
-                        'height levels')
-    parser.add_argument('winddir_filepath', metavar='WINDDIR_FILEPATH',
-                        help='Full path to input NetCDF file of wind direction'
-                        ' on height levels')
-    parser.add_argument('orography_filepath', metavar='OROGRAPHY_FILEPATH',
-                        help='Full path to input NetCDF high resolution '
-                        'orography ancillary. This should be on the same or a '
-                        'finer resolution grid than the input variables, and '
-                        'defines the grid on which the orographic enhancement '
-                        'will be calculated.')
-    parser.add_argument('output_dir', metavar='OUTPUT_DIR', help='Directory '
-                        'to write output orographic enhancement files')
-    parser.add_argument('--boundary_height', type=float, default=1000.,
-                        help='Model height level to extract variables for '
-                        'calculating orographic enhancement, as proxy for '
-                        'the boundary layer.')
-    parser.add_argument('--boundary_height_units', type=str, default='m',
-                        help='Units of the boundary height specified for '
-                        'extracting model levels.')
-
-    args = parser.parse_args(args=argv)
-
-    constraint_info = (args.boundary_height, args.boundary_height_units)
-
-    temperature = load_and_extract(args.temperature_filepath, *constraint_info)
-    humidity = load_and_extract(args.humidity_filepath, *constraint_info)
-    pressure = load_and_extract(args.pressure_filepath, *constraint_info)
-    wind_speed = load_and_extract(args.windspeed_filepath, *constraint_info)
-    wind_dir = load_and_extract(args.winddir_filepath, *constraint_info)
-
-    # load high resolution orography
-    orography = load_cube(args.orography_filepath)
-
-    orogenh_high_res = process(
-        temperature, humidity, pressure, wind_speed, wind_dir, orography)
-
-    # generate file names
-    fname_high_res = os.path.join(
-        args.output_dir, generate_file_name(
-            orogenh_high_res,
-            parameter="orographic_enhancement_high_resolution"))
-
-    # save output files
-    save_netcdf(orogenh_high_res, fname_high_res)
-
-
-def process(temperature, humidity, pressure, wind_speed, wind_dir, orography):
+    raise ValueError('No data available at height {}{}'.format(
+        height_value, units))
+
+
+@cli.clizefy
+@cli.with_output
+def process(temperature: cli.inputcube,
+            humidity: cli.inputcube,
+            pressure: cli.inputcube,
+            wind_speed: cli.inputcube,
+            wind_dir: cli.inputcube,
+            orography: cli.inputcube,
+            *,
+            boundary_height: float = 1000.0, boundary_height_units='m'):
     """Calculate orograhpic enhancement
```

Comment:
I know this isn't your error but there is a typo in orographic here which you could fix.

Reply:
To be fai r this is probably my error from 5 months ago.